package/gluon-hoodselector/luasrc/usr/lib/lua/hoodselector/util.lua

```diff
@@ -0,0 +1,80 @@
+local util = require ('gluon.util')
+local math_polygon = require('math-polygon')
+local json = require ('jsonc')
+local uci = require('simple-uci').cursor()
+local site = require ('gluon.site')
+local logger = require('posix.syslog')
+local M = {}
+
+function M.log(msg)
+	io.stdout:write(msg..'\n')
+	logger.openlog(msg, logger.LOG_PID)
+end
+
+function M.get_domains()
+	local list = {}
+	for _, domain_path in ipairs(util.glob('/lib/gluon/domains/*.json')) do
+		table.insert(list, {
+			domain_code = domain_path:match('([^/]+)%.json$'),
+			domain = assert(json.load(domain_path)),
+		})
+	end
+	return list
+end
+
+-- Return the default domain from the domain list.
+-- This method can return the following data:
+-- * default domain
+function M.get_default_domain(jdomains)
+	for _, domain in pairs(jdomains) do
+		if domain.domain_code == site.default_domain() then
+			return domain
+		end
+	end
+end
+
+-- Get Geoposition.
+-- This method can return the following data:
+-- * table {lat, lon}
+-- * nil for no position
```

Comment:
is `nil` still a possible outcome of the implementation below?

Reply:
fixed

Reply:
this comment seems to be from an older implementation...